index.html

```diff
@@ -12702,7 +12703,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<tbody>
 					<tr>
 						<th class="property-related-head" scope="row">Related Concepts:</th>
-						<td class="property-related"><code>required</code> attribute in [[HTML]]</td>
+						<td class="property-related"><code>[^input/required^]</code> attribute in HTML</td>
```

Comment:
We’ve got a few references to form-related attributes like `required` that could be misinterpreted as _global_ attributes when they actually have multiple distinct HTML spec definitions, like `[^input/required^]` vs. `[^textarea/required^]` vs. `[^select/required^]`.  For now, I just linked to the `input`-scoped definition and didn’t add any additional language to disambiguate.  What do you think?  Here are the applicable attributes I encountered:

- `placeholder`
- `readonly`
- `required`